packages/dev/core/src/Events/deviceInputEvents.ts

```diff
@@ -221,7 +224,7 @@ export interface IMouseEvent extends IUIEvent {
  * Native friendly interface for PointerEvent Object
  */
 export interface IPointerEvent extends IMouseEvent {
-    // Properties
+    inputIndex: Exclude<PointerInput, PointerInput.Horizontal | PointerInput.Vertical | PointerInput.MouseWheelX | PointerInput.MouseWheelY | PointerInput.MouseWheelZ>;

```

Comment:
nit: maybe add comment like IMouseEvent and no newline after (same for IWheelEvent)

Reply:
Fixed